### PR TITLE
35 - Allow customers to exit app

### PR DESCRIPTION
**Pull request by juliano60**
@michowski @ZacTredger Could you please review?

Thanks and Regards,
Jules

### CODE REVIEW

File: src/test/java/newbank/TestApp.java [hunk 1]
```diff
@@ -195,4 +195,11 @@ public void canHandleInvalidDepositAccountOrAmount() throws IOException {
     response = testCommand("DEPOSIT Savings -500.0\n");
     assertThat(response, containsString("FAIL"));
   }
+
+  @Test
+  public void canExitApplication() throws IOException, InterruptedException {
+    String response = testCommand("QUIT\n");
+    assertThat(response, containsString("SUCCESS"));
```
Comment (michowski) [inline src/test/java/newbank/TestApp.java:202]: Curious if we could (easily) assert that the thread has terminated too.

**Review by michowski — APPROVED** [review]
AWESOME!

File: src/test/java/newbank/TestApp.java (hunk 1 above)
Comment (jkonan100) [reply src/test/java/newbank/TestApp.java:202]: This is what the "client.join()" does.

**Comment by juliano60** [comment]
Cheers buddy.

Kindest Regards,
Jules

**Pull request merged**